int16 battery_level
int16 battery_voltage
int16 battery_current
int16 battery_temperature
uint8 supply_alarms